reporter/src/main/kotlin/utils/StatisticsCalculator.kt

```diff
@@ -92,9 +92,11 @@ internal class StatisticsCalculator {
         ortResult
             .getProjects()
             .filterNot { project -> ignoreExcluded && ortResult.isExcluded(project.id) }
-            .flatMap { project -> project.scopes }
-            .filterNot { scope -> ignoreExcluded && ortResult.repository.config.excludes.isScopeExcluded(scope) }
-            .map { scope -> scope.getDependencyTreeDepth() }
+            .flatMap { project ->
+                val scopes = ortResult.dependencyNavigator.scopeNames(project)
+                MutableList(scopes.size) { project }.zip(scopes)
+            }.filterNot { (_, scope) -> ignoreExcluded && ortResult.repository.config.excludes.isScopeExcluded(scope) }
```

Comment:
I'm not sure, but didn't you introduce a function for the excluded check to the dependency navigator in a previous commit?

Reply:
I only added an overloaded function to `Excludes`, which accepts a scope name rather than a full `Scope`.